src/docs/product/dashboards/index.mdx

```diff
@@ -16,13 +16,22 @@ If you’re interested in being an Early Adopter, you can turn your organization
 
 </Note>
 
-Dashboards provide the ability to build custom views of your application's health, including the ability to navigate issues and performance across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
+Dashboards will enable you to get a broad overview of your application’s health. You’ll be able to navigate through error and performance data across multiple projects and zoom in on any time series visualizations you may want to investigate. All widgets in the same view will update synchronously to reflect the new date range in the Global Selection Header.
 
-![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard.png)
+![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard_default.png)
 
-You can filter the information that's displayed in a dashboard in different ways:
+## Default View
+The view you’ll start out with will contain the following widgets:
 
-- Global Selection Header: Specify which projects, environments, and date range you want to zoom in on at the top of the page. This can also be found in other parts of [sentry.io](https://sentry.io) as a top-level filter.
-- Interactive Graphs: Use the interactive graph function of time series visualizations to zoom in. To do this, click and drag over the area you want to investigate.
+- Number of Errors - Shows the total numbers of errors you have.
+- Number of Issues - Shows the total number of issues you have.
+- All Events - Shows all errors, transactions, CSP, and more.
+- Affected Users - Compares Known Users v. Unknown Users.
+- Handled v. Unhandled - Compares Handled Errors v. Unhandled Errors.
+- Errors by Country - Shows a world map of error frequency.
+- Errors by Browser - Compares number of errors by browser name.
```

Comment:
Minor edits. Should spell out what CSP means though.

```suggestion
- Number of Errors: Shows the total numbers of errors you have.
- Number of Issues: Shows the total number of issues you have.
- All Events: Shows all errors, transactions, CSP, and more.
- Affected Users: Compares Known Users to Unknown Users.
- Handled v. Unhandled: Compares Handled Errors to Unhandled Errors.
- Errors by Country: Shows a world map of error frequency.
- Errors by Browser: Compares the number of errors by browser name.
```

Reply:
Changed